htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -2144,7 +2144,7 @@ public function addline($desc, $pu, $txtva, $txlocaltax1, $txlocaltax2, $qty, $f
 				$pu = 0;
 			}
 
-			$localtaxes_type = getLocalTaxesFromRate($txtva, 0, $mysoc, $this->thirdparty);
+			$localtaxes_type = getLocalTaxesFromRate($txtva, 0, $mysoc, $mysoc);
```

Comment:
what is wrong with old code ?
Param 3 must be the buyer, and param 4 must be the seller (so the thirdparty).

Reply:
Yes, I was wrong, I miss read $myssoc as first parameters